modules/tableview/cells/multi-line-left-detail.js

```diff
@@ -23,7 +25,13 @@ export class MultiLineLeftDetailCell extends React.PureComponent<LeftDetailProps
 				</Text>
 			</View>
 		)
-		return <Cell cellContentView={cellContent} />
+		return (
+			<Cell
+				accessory={accessory}
+				cellContentView={cellContent}
+				onPress={onPress ? onPress : null}
```

Comment:
If you've defined `onPress` to be potentially null, why the runtime check here?

Reply:
To prevent the cell from being pressable

Reply:
Ah — I see your point now. I was thinking about a different file.